tests/modelgauge_tests/test_safe.py

```diff
@@ -321,3 +316,12 @@ def test_registered_tests(self):
             for locale in locales:
                 uid = f"safe-{hazard}-{locale}-1.0"
                 assert TESTS._lookup.get(uid)
+
+    def test_uid_construction(self):
+        # locale variants
+        assert BaseSafeTestVersion1.create_uid("xyz", "en_US") == "safe-xyz-en_us-1.0"
+        assert BaseSafeTestVersion1.create_uid("xyz", Locale.EN_US) == "safe-xyz-en_us-1.0"
```

Comment:
Should we consider updating the `Locale` enum to use mixed-case language strings?

Reply:
Ah, agreed. Since we both want it, I'll do that now.